gulpfile.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+
+gulp.task('lint', () => {
+  gulp.src(['./simple-obj-test-1/**/*.js', 'gulpfile.js'])
+    .pipe(eslint())
```

Comment:
I get failures running the lint tests in gulp.  You might want to add a rule to allow es6: {
      parserOptions: {
        "ecmaVersion": 6
      } to the lint call 
eslint({
      parserOptions: {
        "ecmaVersion": 6
      })